.clang-tidy

```diff
@@ -82,6 +82,9 @@ Checks: '*,
 -readability-static-accessed-through-instance,
 # we are okay with lower case,
 -readability-uppercase-literal-suffix,'
+CheckOptions:
+  - key: performance-move-const-arg.CheckTriviallyCopyableMove
+    value: 0
```

Comment:
[optional] [The docs](https://clang.llvm.org/extra/clang-tidy/checks/performance/move-const-arg.html) say to use *true* to enable it, so should we use *false* to disable it? Or is it more standard to use `0`? I wasn't able to find anything after a quick search online.

Reply:
Looks like they changed it at some point, probably version 11 or 12 is my guess.  We're about to drop support for those anyway (and it's not clear whether that was even a behavior change or just a doc change), so I'll switch to the new form.

Reply:
Looks like it changed in [clang-12](https://releases.llvm.org/12.0.1/tools/clang/tools/extra/docs/clang-tidy/checks/performance-move-const-arg.html) because [clang-11](https://releases.llvm.org/11.0.1/tools/clang/tools/extra/docs/clang-tidy/checks/performance-move-const-arg.html) has the old method.